go.mod

```diff
@@ -12,13 +12,17 @@ require (
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/crypto v0.11.0
 	golang.org/x/exp v0.0.0-20230801115018-d63ba01acd4b
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230807174057-1744710a1577
+	google.golang.org/grpc v1.57.0
```

Comment:
Please use 1.56.2 which is the version currently used in the runtime, or we'll force the runtime to upgrade grpc late in the game: https://github.com/dapr/dapr/blob/ad5618711830510617e1b8fb2d0c6758f14ff6b9/go.mod#L67C3-L67C3